### DISCUSSION TITLE
List of animatable and not animatable CSS properties

### DISCUSSION THREAD

**Discussion by Vallek** [mdn/mdn-community#400] [Content]
Hello everyone! I recreated lists of animatable and not animatable CSS properties instead of the one removed from MDN from w3c api. Because I preferred to look them up on one page) 

If this is something you would also use you can check it here https://vallek.github.io/animatable-css/

**Comment by bsmth** [comment]
Thanks a lot for sharing. I'd also link here for further discussion about this: https://github.com/mdn/content/issues/28554
